### ISSUE TITLE
dashboard no show

### ISSUE THREAD

**Issue by chanwoner** [ylasgamers/getgrass#10]
![微信图片_20241213132853](https://github.com/user-attachments/assets/c0e9fd3b-3e05-4184-aa2f-e0238981fc42)

**Comment by chanwoner** [comment]
![微信截图_20241213133210](https://github.com/user-attachments/assets/759e6367-bc29-4ad9-849a-acdac1f39b95)

**Comment by Trillz007** [comment]
> ![微信截图_20241213133210](https://private-user-images.githubusercontent.com/38780082/395441837-759e6367-bc29-4ad9-849a-acdac1f39b95.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQxMjg2NzQsIm5iZiI6MTczNDEyODM3NCwicGF0aCI6Ii8zODc4MDA4Mi8zOTU0NDE4MzctNzU5ZTYzNjctYmMyOS00YWQ5LTg0OWEtYWNkYWMxZjM5Yjk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEzVDIyMTkzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQxZTI3ZWY2MWVlMWMzNDM5YjkzY2U5MmUxZmYyMDE2YWY5MzhmZjhjMmNhNDAzY2YzNmYwYTUxMzIwYmEzYzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.uqaQXyg1RdPr9WhYunQ-Si1lglos7kNBDo9FWPSuc9k)

Did your points add up later or your proxy bandwidth got wasted?

**Comment by Bekloppten** [comment]
have the problem too.i didnt get points for days.
i tried everything,all bots i found, different vps, old, medium and brandnew accounts. nothing works for me suddenly.
dont you have all problems at the moment ? if not, how can i find the reason, why it doesnt work ?

**Comment by Solana0x** [comment]
yeah This issue is known probably grass team reinforced some sort of Auth Mech or protection

**Comment by Bekloppten** [comment]
Man, its you! 
Thank you for all the great bots! I used a lot of your work. 
i thought one of your bots, was written by smb else. but after reviewing the files, im pretty sure, you created them.
do you maybe got a telegram channel or a place with other like minded, where we could talk ?

**Comment by rahmap** [comment]
i think this method is fixed by Grass, so bot wont work again

**Comment by YiMoWanXia** [comment]
I only heard that grass added an http request, but why can't I receive the 'action' of 'HTTP_REQUEST' when I initiate a websocket connection?
